book/src/asm/operations.md

```diff
@@ -0,0 +1,13 @@
+# Operations
```

Comment:
There is a case for links in VMs: the program lookup can be seen as an unconditional link. This would be used only internally now and I'm not sure how to expose it to the user yet. For operations I can't see a case right now and it seems unlikely there will be one.